features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth_response.html

```diff
@@ -62,27 +61,30 @@
         .loader-redirect {
             font-size: 17px;
             color: #787778;
+            margin-right: 0.5rem;
         }
+
+        .primary-color-btn {
+            color: var(--primary-color) !important;
+            cursor: pointer;
+        }
+
     </style>
 </head>
 
-<body onload="javascript:document.forms[0].submit()">
-
+<body onload="javascript:document.getElementById('oauth-response').submit()">
     <div class="padding-top-20">
         <div class="ui centered grid">
             <div class="twelve wide mobile eight wide tablet six wide computer column">
                 <div class="redirect-loader">
                     <i class="fa fa-circle-o-notch fa-spin loader-redirect"></i>
                     Please wait while we take you to the application.
                 </div>
-                <p class="padding-top-2 padding-bottom-1 text-align-center">Click continue if you have been waiting for
+                <p class="padding-bottom-1 text-align-center">Click <span class="primary-color-btn button" onclick="javascript:document.getElementById('oauth-response').submit()">here</span> if you have been waiting for
```

Comment:
Shall we use an anchor tag here in stead wrapping up with a span tag?

Reply:
Fixed